src/Symfony/Component/Cache/Adapter/TagAware/AbstractTagAwareAdapter.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Cache\Adapter\TagAware;
```

Comment:
For all right?

On one side I was thinking TagAware is separate namespace side by side with PSR6 (Adapter) and PSR16 (Simple), but since it's not standardized (yet), I opted for this to organize it in one namespace.

But if you want it all in Adapter I'm fine with that.

Reply:
all in Adapter is fine to me

Reply:
@nicolas-grekas Should I then perhaps change naming to be like `TagAware[Native]RedisAdapter` or similar?

Reply:
Solved

Kept the names as is, if you prefer what I proposed above, then open new review point. I'll happily adapt.